cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -86,8 +86,8 @@ public TestCommand(Path projectPath, PrintStream outStream, PrintStream errStrea
             "dependencies.")
     private boolean offline;
 
-    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve dependencies.")
-    private boolean skipLock;
+//    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve dependencies.")
```

Comment:
```suggestion
//    @CommandLine.Option(names = {"--skip-lock"}, description = "Skip using the lock file to resolve the dependencies.")
```

Reply:
This is a commented code block, hence updating is not required.